src/actor/mod.rs

```diff
@@ -36,6 +36,29 @@ fn actor_claims(wasm: impl AsRef<[u8]>) -> Result<jwt::Claims<jwt::Actor>> {
     Ok(claims.claims)
 }
 
+/// Returns true if the host call is authorized and false otherwise
+#[instrument]
+fn authorized_call(
+    claims: &jwt::Claims<jwt::Actor>,
+    binding: &str,
+    namespace: &String,
+) -> Result<()> {
+    if binding.is_empty() && namespace.starts_with('M') && namespace.len() == 56 {
+        // This is an actor-to-actor invocation, skip capability check
+        return Ok(());
+    }
```

Comment:
Quick check @autodidaddict , will the namespace be populated with the real actor ID here if I use a call alias?